Dockerfile

```diff
@@ -23,24 +23,30 @@
 
 # The Ubuntu image that should be used as the basis for the guacd image
 ARG UBUNTU_BASE_IMAGE=21.10
-
 # Use Debian as base for the build
 FROM ubuntu:${UBUNTU_BASE_IMAGE} AS builder
 
+ARG USE_APT_REPOSITORY_MIRROR=true
+# More faster apt repo
+ARG APT_SOURCE="http://archive.ubuntu.com/ubuntu"
+ARG APT_SECURITY_SOURCE="http://security.ubuntu.com/ubuntu"
 #
 # The Debian repository that should be preferred for dependencies (this will be
 # added to /etc/apt/sources.list if not already present)
 #
 # NOTE: Due to limitations of the Docker image build process, this value is
 # duplicated in an ARG in the second stage of the build.
 #
-ARG UBUNTU_RELEASE=impish-backports
+ARG UBUNTU_RELEASE=impish
```

Comment:
Yes , that is  the design of software  backports  version . The backports  version contains some new feature . Sorry , I haven't know this point.

Reply:
But I have a question . Because I don't see specify version in Dockerfile .  I think  if  the version is not specified , then backports version is always be considered as a backup.  Do you mean this two packages ?
![image](https://user-images.githubusercontent.com/30000877/164951535-abbf9c4d-f85a-462c-894b-739100845dfd.png)

Reply:
And I have tried delete backports repo line in /etc/apt/sources.list ,it still can be installed success. Here is a screenshot  
![image](https://user-images.githubusercontent.com/30000877/164951734-d09a9de1-1e89-4cf8-a3b7-4ec09aedc190.png)

Reply:
So maybe  these feature you menthioned above is include in main version ?

Reply:
After search in this wiki  
https://help.ubuntu.com/community/UbuntuBackports 
You are right, There is -t argument in apt install command.